test/cmd/basicresources.sh

```diff
@@ -282,6 +282,18 @@ echo "OAuth descriptor: ok"
 
 os::cmd::expect_success 'oc delete all --all'
 
+os::test::junit::declare_suite_start "cmd/basicresources/get"
```

Comment:
also, control the environment in that new script... there are three envvars we care about: $HOME (because we look in $HOME/.kube/config), $KUBECONFIG, and $KUBERNETES_MASTER. You might want to set HOME to a tmpdir (I think we do this elsewhere) to ensure no ~/.kube/config file pollutes tests, and unset KUBECONFIG and KUBERNETES_MASTER